<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsView</class>
 <widget class="QWidget" name="StatisticsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>统计与同步管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutStats">
     <item>
      <widget class="QGroupBox" name="groupBoxDaily">
       <property name="title">
        <string>今日统计</string>
       </property>
       <layout class="QFormLayout" name="formLayoutDaily">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>今日就诊人数：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lbTodayPatients">
          <property name="text">
           <string>0</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #2196F3;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>今日预约数：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbTodayAppointments">
          <property name="text">
           <string>0</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #4CAF50;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>今日处方数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lbTodayPrescriptions">
          <property name="text">
           <string>0</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #FF9800;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>今日收入：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lbTodayRevenue">
          <property name="text">
           <string>0.00</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #E91E63;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxMonth">
       <property name="title">
        <string>本月统计</string>
       </property>
       <layout class="QFormLayout" name="formLayoutMonth">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>本月就诊人数：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lbMonthPatients">
          <property name="text">
           <string>0</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #2196F3;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>本月预约数：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbMonthAppointments">
          <property name="text">
           <string>0</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #4CAF50;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>本月处方数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lbMonthPrescriptions">
          <property name="text">
           <string>0</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #FF9800;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>本月收入：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lbMonthRevenue">
          <property name="text">
           <string>0.00</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #E91E63;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxReport">
     <property name="title">
      <string>报表生成</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutReport">
      <item>
       <widget class="QPushButton" name="btDailyReport">
        <property name="text">
         <string>生成日报表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btMonthlyReport">
        <property name="text">
         <string>生成月报表</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSync">
     <property name="title">
      <string>数据同步</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutSync">
      <item>
       <widget class="QPushButton" name="btSyncDrugs">
        <property name="text">
         <string>同步药品信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btSyncDiagnoses">
        <property name="text">
         <string>同步诊断标准</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btSyncAll">
        <property name="text">
         <string>同步所有数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLog">
     <property name="title">
      <string>操作日志</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutLog">
      <item>
       <widget class="QTextEdit" name="teLog">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
